src/App.js

```diff
@@ -0,0 +1,30 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import Navbar from './components/Navbar';
+import MyProfile from './components/MyProfile';
+import Rockets from './components/Rockets';
+import Missions from './components/Missions';
+import { getMissions } from './Redux/missions/missionsSlice';
+import { fetchData } from './Redux/Rockets/rocketSlice';
+
+function App() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchData());
+    dispatch(getMissions());
```

Comment:
- [ ] Good job on fetching the Rockets when the application starts, however, you are also fetching the missions. Please kindly ensure you only fetch missions once when you navigate to the missions page. You can sort this out by moving the dispatch action to the missions component.

Reply:
Addressed